container-scan/action.yaml

```diff
@@ -0,0 +1,98 @@
+name: "Security Scan Composite Action"
+description: "GitHub Action for scanning container image for vulnerabilities using Lacework"
+
+inputs:
+  docker-config-file:
+    required: false
+    description: Path to the docker config file (defaults to .docker-config.json) Must contain imageName, may contain dockerfile
+    default: .docker-config.json
+  github-token:
+    required: true
+    description: GitHub token
+  lw-account-name:
+    required: true
+    description: Lacework account name
+  lw-access-token:
+    required: true
+    description: Lacework access token
+  image-tag:
+    required: true
+    description: Docker image tag
+
+outputs:
+  comment-id:
+    description: Comment ID of the test report
+    value: ${{ steps.comment-pr.outputs.comment-id }}
+  image-name:
+    description: Docker image name
+    value: ${{ steps.config.outputs.image-name }}
+
+runs:
+  using: composite
+  steps:
+    - name: Checkout Repository
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
```

Comment:
I think for this case this could be removed, since we only need the curent commit and not the history, right?